applications/vessel-history/src/features/vessel-list-item/VesselListItem.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('<VesselListItem />', () => {
     otherShipnames: [],
     shipname: 'DON TITO',
     source: 'AIS',
-    dataset: 'public-global-vessels:v20190502',
+    dataset: 'public-global-fishing-vessels:v20201001',
```

Comment:
Is this a unit test that mocks the API or you are actually calling the service?
If it is a mock you could change this to `vYYYYMMDD` so it is clear it is a mock.